### PR TITLE
added method to populate chess board with chess pieces

### PR DESCRIPTION
**Pull request by harrisonlim**
Err I hope this is correct. Good to fair chance that 'color: :black' is not the right way to denote the piece color. Also, not sure if I need to differentiate the different similar pieces, like the multiple pawns or say pawns belonging to either white/black. CCW

### CODE REVIEW

**Comment by varghes1** [comment]
i think to start off - based on what Jackie is doing you might need to do ChessPiece.create since all the pawns/rooks/knights/etc will be inheriting from that. and then you can do type: rook/pawn/etc.

I may be wrong but you need to include game_id also? to show which game the pieces belong to.

**Comment by KaoruDev** [comment]
@varghes1 what @harrisonlim has is actually correct. When you do `Pawn.create` `Rails` will automatically fill in the `type` property of the model as a `Pawn` Magic of STI =)

**Comment by KaoruDev** [comment]
@varghes1 tho you are correct about the `game_id`, these pieces need `belongs_to` a the game that is creating them.

File: app/models/game.rb [hunk 1]
```diff
@@ -2,4 +2,40 @@ class Game < ActiveRecord::Base
...
+  	Knight.create(x_position: 0, y_position: 6, color: :black)
+  	Rook.create(x_position: 0, y_position: 7, color: :black)
+  	Pawn.create(x_position: 1, y_position: 0, color: :black)
+  	Pawn.create(x_position: 1, y_position: 1, color: :black)
+  	Pawn.create(x_position: 1, y_position: 2, color: :black)
+  	Pawn.create(x_position: 1, y_position: 3, color: :black)
+  	Pawn.create(x_position: 1, y_position: 4, color: :black)
+  	Pawn.create(x_position: 1, y_position: 5, color: :black)
+  	Pawn.create(x_position: 1, y_position: 6, color: :black)
+  	Pawn.create(x_position: 1, y_position: 7, color: :black)
```
Comment (KaoruDev) [inline app/models/game.rb:22]: is there a way we can `DRY` up the Pawn creation?

**Pull request closed without merging**